packages/EasyApiToken/src/Interfaces/ApiTokenDecoderInterface.php

```diff
@@ -23,5 +23,3 @@ public function decode($request): ?ApiTokenInterface;
 
     public function getName(): string;
 }
-
-\class_alias(ApiTokenDecoderInterface::class, EasyApiTokenDecoderInterface::class);
```

Comment:
This easily messes up PHPStan and Rector. Why is this here BTW?

If it's legacy interface, it should be here to inherit from this one